docs/build-on-polkadot/xcm/hrmp-channels/para-to-para.md

```diff
@@ -0,0 +1,181 @@
+![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/hrmp-channels-2.webp)
+
+??? note "Calculating Parachain Sovereign Account"
+    To generate the sovereign account address for a parachain, you'll need to follow these steps:
+
+    1. Determine if the parachain is an "up/down" chain (parent or child) or a "sibling" chain:
+
+        - Up/down chains use the prefix `0x70617261` (which decodes to `b"para"`)
+
+        - Sibling chains use the prefix `0x7369626c` (which decodes to `b"sibl"`)
+
+    2. Calculate the u32 scale encoded value of the parachain ID:
+        - Parachain 2500 would be encoded as `c4090000`
+
+    3. Combine the prefix and parachain ID encoding to form the full sovereign account address:
+
+        The sovereign account of parachain 2500 in relay chain will be `0x70617261c4090000000000000000000000000000000000000000000000000000`
+        and the SS58 format of this address is `5Ec4AhPSY2GEE4VoHUVheqv5wwq2C1HMKa7c9fVJ1WKivX1Y`
+    
+    To perform this conversion, you can also use the **"Para ID" to Address** section in [Substrate Utilities](https://www.shawntabrizi.com/substrate-js-utilities/).
+
+### Step 2 - Create Channel Opening Extrinsic
+
+1. In Polkadot.js Apps, connect to the relay chain, navigate to the **Developer** dropdown and select the **Extrinsics** option
+
+    ![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/para-to-para/hrmp-para-to-para-1.webp)
+
+2. Construct an `hrmpInitOpenChannel` extrinsic call
+
+    1. Select the **hrmp** pallet
+    2. Choose the **hrmpInitOpenChannel** extrinsic
+    3. Fill in the parameters
+        - **recipient** - parachain ID of the target chain (in this case, 2600)
+        - **proposedMaxCapacity** - max number of messages that can be pending in the channel at once
+        - **proposedMaxMessageSize** - max message size that could be put into the channel
+    4. Copy the encoded call data
+    ![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/para-to-para/hrmp-para-to-para-2.webp)
+    The encoded call data for opening a channel with parachain 2600 is `0x3c00280a00000800000000001000`.
+
+### Step 3 - Crafting and Submitting the XCM Message from the Sender
+
+To initiate the HRMP channel opening process, you need to create an XCM message that includes the encoded `hrmpInitOpenChannel` call data from the previous step. This message will be sent from your parachain to the relay chain.
+
+This example uses the `sudo` pallet to dispatch the extrinsic. Verify the XCM configuration of the parachain you're working with and ensure you're using an origin with the necessary privileges to execute the `polkadotXcm.send` extrinsic.
+
+The XCM message should contain the following instructions:
+
+- **WithdrawAsset** - withdraws assets from the origin's ownership and places them in the Holding Register
```

Comment:
```suggestion
- **`WithdrawAsset`** - withdraws assets from the origin's ownership and places them in the Holding Register
```
Are these param names or similar? That's why I'm suggesting backticks. If not, disregard.